docs/guides/developer/docs/release-manager.md

```diff
@@ -1,474 +1,377 @@
+6. If everything looks fine, commit the changes and push it to the community repository:
 
-        git commit -as
+        git add $(git status | grep 'modified:.*pom.xml' | awk '{print $2;}')
+        git commit -s
         git push <remote> develop
 
-9. Finally change the `fixVersion` and `summary` on the merge ticket for 3.0 and create a new one for develop.
-
-
-At this point, the developer community should then be notified. Consider using the following as a template email:
-
-    To: dev@opencast.org
-    Subject: Release Branch for Opencast <VERSION> Cut
-
-    Hi everyone,
-    it is my pleasure to announce that the <VERSION> release branch has now been
-    cut, available as r/<VERSION>. Pull requests for bug fixes are to be made
-    against this branch.
 
-    Remember that the release schedule for this release:
+At this point, the developer community should then be notified by writing an email like the following to the developers
+list:
 
-      <RELEASE_SCHEDULE>
+```no-highlight
+To: dev@opencast.org
+Subject: Release Branch for Opencast <version> Cut
 
-    As always, we hope to have a lot of people testing this version, especially
-    during the public QA phase.  Please report any bugs or issues you encounter.
+Hi everyone,
+the Opencast <version> release branch (r/<version>) has been
+cut.  Pull requests for bug fixes may still be made against
+this branch but, as usual, features should go into develop
+instead.
 
+Remember the release schedule for this release:
 
-### Check Status of Language Translations
+  <release_schedule>
 
-On the date when the release branch is cut, the release manager is responsible to check whether there are language
-translations that need to be included or excluded for the upcoming release.
-Have a look at [Inclusion and Exclusion of Languages](localization.md) for the criteria.
+As always, we hope to have a lot of people testing this
+version, especially during the public QA phase. Please
+report any bugs or issues you encounter.
 
-1. Check whether translations not yet included in Opencast meet the inclusion criteria (candidates for includsion)
-2. Check whether translations in Opencast meet the exclusion criteria (endangered translations)
-3. Publish an announcement on the Opencast Users list that specifies:
-    a. Translations that will be included in the upcoming release
-    b. Endangered translations
+For testing, you may use https://stable.opencast.org if you
+do not want to set up a test server yourself. The server is
+reset on a daily basis and will follow the new release
+branch with its next rebuild.
 
-Please create a single post on the Opencast Users list:
+Additionally, look out for announcements regarding container
+and package builds for testing on list if you want to run
+your own system but do not want to build Opencast from
+source.
+```
 
-    To: users@opencast.org
-    Subject: Opencast <VERSION>: Language translation status
 
-    Hi everyone,
+### Status of Translations
```

Comment:
What about regular updates about the translation status? The release manager should also point the various ways to contribute to the translations:

* translate strings
* verify translations
* help clear up issues of fellow translators

Reply:
I think that's a good idea but where would you put this information? Right now, we are sending out several emails. Would you suggest to include it in all of the emails? Or send out an additional mail? Or send all information (languages endangered, languages included and how to contribute) in a single email instead?

Reply:
As was done in the past, I regularly sent reminder status update mails ([1](https://groups.google.com/a/opencast.org/forum/?fromgroups=#!topic/users/bH8vpkaVMJo), [2](https://groups.google.com/a/opencast.org/forum/?fromgroups=#!topic/users/VwOxqB-8l_o), [3](https://groups.google.com/a/opencast.org/forum/?fromgroups=#!topic/users/mSPhAFxT8fc), [4](https://groups.google.com/a/opencast.org/forum/?fromgroups=#!topic/users/ynhbT2BmNdM)) so that would be an option. The [translation week mail](https://groups.google.com/a/opencast.org/forum/?fromgroups=#!topic/users/Kjp2WB4GAI0) is another one. In this example only the "help clear up issues of fellow translators" is missing.